Math/src/main/java/org/tribuo/math/util/VectorNormalizer.java

```diff
@@ -43,4 +48,21 @@ default public void normalizeInPlace(double[] input) {
         }
     }
 
+    /**
+     * Exports this normalizer as a list of ONNX NodeProtos.
+     * <p>
+     * For compatibility reasons the default implementation returns an empty list, and
+     * normalizers which use the default implementation will not be able to export
+     * their models to ONNX. It is recommended that this method is overridden to
+     * support ONNX export, and in a future version of Tribuo this default implementation
+     * will be removed.
+     * @param context The ONNX context object for name generation.
+     * @param input The name of the input to normalize.
+     * @param output The name of the normalized output.
+     * @return A list of node protos representing the normalization operation.
+     */
+    default public List<OnnxMl.NodeProto> exportNormalizer(ONNXContext context, String input, String output) {
+        return Collections.emptyList();
```

Comment:
Since it is never correct to call this method directly, it should at least log a warning.

Reply:
Sure.